synapse/tests/test_lib_stormlib_modelext.py

```diff
@@ -57,12 +57,24 @@ async def test_lib_stormlib_modelext_base(self):
             model_defs = await core.callStorm('return ( $lib.model.ext.getExtModel() )')
             self.isinstance(model_defs, dict)
 
+            self.len(1, await core.nodes('_visi:int:tick'))
```

Comment:
I'll add a quick abandoned property test here once the other questions I have raised are addressed.